test/shmemx/Makefile.am

```diff
@@ -31,26 +31,10 @@ check_PROGRAMS += \
 	atomic_nbi \
 	put_signal \
 	put_signal_nbi \
-	shmemx_wait_until_all \
-	shmemx_wait_until_any \
-	shmemx_wait_until_some \
```

Comment:
`shmemx_wait_until_some` also looks like it uses `shmem_p` instead of `shmem_atomic_set`...